programs/programs/co/universal_preschool/calculator.py

```diff
@@ -23,16 +26,20 @@ def member_value(self, member: HouseholdMember):
         qualifying_condition = self._has_qualifying_condition(member)
 
         if not qualifying_condition:
-            return UniversalPreschool.amount_by_hours["15_hours"]
+            return UniversalPreschool.amount_15_hr
 
         if member.age == UniversalPreschool.age:
-            return UniversalPreschool.amount_by_hours["30_hours"]
+            return UniversalPreschool.amount_30_hr
 
-        return UniversalPreschool.amount_by_hours["10_hours"]
+        return UniversalPreschool.amount_10_hr
 
     def _has_qualifying_condition(self, member: HouseholdMember):
-        fpl = self.program.fpl.as_dict()
-        income_limit = int(UniversalPreschool.percent_of_fpl * fpl[self.screen.household_size])
-        income_condition = self.screen.calc_gross_income("yearly", ["all"]) < income_limit
+        fpl = self.program.fpl.as_dict()[self.screen.household_size]
+        income = self.screen.calc_gross_income("yearly", ["all"])
 
-        return income_condition or member.relationship == "fosterChild"
+        income_limit = int(UniversalPreschool.income_limit * fpl)
+
+        if member.relationship == "fosterChild":
+            income_limit = int(UniversalPreschool.foster_income_limit * fpl)
+
+        return income < income_limit
```

Comment:
Does the `income` have to be strictly less than the `income_limit` or can it also be equal to the `income_limit`?

Reply:
I don't know. I just copied what we had previously. I am going to update it to allow it to be equal to because that is what we usually do.